be/src/http/action/update_config_action.cpp

```diff
@@ -90,18 +90,27 @@ Status UpdateConfigAction::update_config(const std::string& name, const std::str
             return Status::OK();
         });
         _config_callback.emplace("storage_page_cache_limit", [&]() -> Status {
-            ASSIGN_OR_RETURN(int64_t cache_limit, DataCache::GetInstance()->get_storage_page_cache_limit());
-            cache_limit = DataCache::GetInstance()->check_storage_page_cache_limit(cache_limit);
-            StoragePageCache::instance()->set_capacity(cache_limit);
+            StoragePageCache* cache = DataCache::GetInstance()->page_cache();
```

Comment:
What's the difference between updating `storage_page_cache_limit` and `datacache_mem_size` now?

Reply:
I've only made this change temporarily because we can no longer configure the PageCache size. However, I haven't decided whether to throw an error directly or switch to modifying DataCache instead. For now, I'll leave it as is. I plan to submit a PR later to change it to: (if block_cache_mem is enabled, throw an error); otherwise, convert it to data_cache_mem_size.